teps/0074-deprecate-pipelineresources.md

```diff
@@ -159,8 +159,9 @@ that maybe this concept, at least in its current form, isn't required).
 1. ~~Mark PipelineResources as deprecated in our Pipelines documentation; announce it officially in the next
    Pipelines release.~~
 1. Update tutorials, examples (except those used for testing), and dogfooding Pipelines to not use PipelineResources.
-1. Continue to support `PipelineResources` in the Pipelines `v1beta1` API for at least 9 months after announcing their deprecation,
-   following our [stability policy](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md).
+1. ~~Continue to support `PipelineResources` in the Pipelines `v1beta1` API for at least 9 months after announcing their deprecation,
+   following our [stability policy](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md).~~ PipelineResources
+   are going [to be removed when swapping the storage version to v1](https://github.com/tektoncd/pipeline/issues/5967).
```

Comment:
So we technically have already supported PipelineResources for 9 months after announcing their deprecation. The `resources` fields of our existing CRDs are beta fields, but we can remove beta fields after 9 months too. It might make sense to just mark this as complete. PipelineResources aren't really related to the storage version of the api so I'd either add a note explaining why now, or remove the references to the storage version.

Reply:
Thanks Lee, updated